syntax = "proto3";
 
package transport_router_proto;
 
message RouterSettings {
    uint32 bus_wait_time = 1;
    double bus_velocity = 2;
}